api/specs.proto

```diff
@@ -206,14 +206,14 @@ message EndpointSpec {
 		// network. The user may then provide an alternate
 		// option for ingress load balancing which is
 		// transparent to swarm.
-		DISABLED = 1 [(gogoproto.enumvalue_customname) = "IngressRoutingDisabled"];
+		LOCAL = 1 [(gogoproto.enumvalue_customname) = "IngressRoutingDisabled"];
```

Comment:
I couldn't recall what `DISABLED` means.

Also, #911 would like to expose ports directly in the engine - do we support that and should THAT be `LOCAL`?